tests/modifiers.edge-split-modifier.test.three-js-test.html

```diff
@@ -510,12 +512,12 @@
 
         import * as THREE from 'three';
 
-        import { OrbitControls } from './js/OrbitControls.js';
-        import { OBJLoader } from './js/OBJLoader.js';
+        import { OrbitControls } from 'three/examples/jsm/controls/OrbitControls.js';
+        import { OBJLoader } from 'three/examples/jsm/loaders/OBJLoader.js';
         import { EdgeSplitModifier } from '@dioxygen-software/three-js-extra-public';
-        import * as BufferGeometryUtils from './js/BufferGeometryUtils.js';
+        import { BufferGeometryUtils } from '../node_modules/three/examples/jsm/utils/BufferGeometryUtils.js';
```

Comment:
Don't load it from `../node_modules` but with a bare import as in previous lines. The import map is already responsible for resolution against `node_module`.